types/types.go

```diff
@@ -53,8 +53,8 @@ type MgmtNet struct {
 
 // NodeConfig is a struct that contains the information of a container element
 type NodeConfig struct {
-	ShortName        string
-	LongName         string
+	ShortName        string // node of the Node inside topology YAML
```

Comment:
**name** of the node

Reply:
haha, yep. btw, while we're at it, how about we rename `ShortName` to something like `TopoName` and `LondName` to `ContainerName`? or something similar to give a bit more context to these variables?

Reply:
m, TopoName to me feels more like the name of the topology
If I were changing these two fields, I'd go maybe with `Name` for a short name and `FullName` or `FQN` for a fully qualified name

@karimra how do you feel about Short/Long name pair and the possible alternatives? For us I guess the context is clear, thus it doesn't mix too much which is which

Reply:
maybe just `Name` and `ContainerName` ?

Reply:
that sounds good @karimra 
we can do this in a separate PR